app/models/saved_search.rb

```diff
@@ -5,4 +5,6 @@ class SavedSearch < ApplicationRecord
   validates :title, uniqueness: { scope: :team_id }, unless: proc { |ss| ss.is_being_copied }
 
   belongs_to :team, optional: true
+  has_many :feeds
+  has_many :feed_teams
```

Comment:
I think we need a `dependent: :nullify` here

Reply:
Done